sdk-webview/src/main/java/com/reach5/identity/sdk/webview/WebViewProvider.kt

```diff
@@ -42,13 +42,14 @@ class ConfiguredWebViewProvider(
         const val RESULT_INTENT_ERROR = "RESULT_INTENT_ERROR"
     }
 
-    override fun login(origin: String, activity: Activity) {
+    override fun login(origin: String, scope: Collection<String>?, activity: Activity) {
         val intent = Intent(activity, ReachFiveLoginActivity::class.java)
         intent.putExtra(
             BUNDLE_ID, WebViewProviderConfig(
                 providerConfig = providerConfig,
                 sdkConfig = sdkConfig,
-                origin = origin
+                origin = origin,
+                scope = providerConfig.scope.plus(scope ?: emptySet()).joinToString(" ")
```

Comment:
Are we sure we want to append the given scopes to all of the existing configured scopes? Currently, it won't be possible to request a narrower scope than already configured.

Reply:
What do you suggest? This logic was validated by @gpartenet.

Reply:
This isn't clearly specified in the ticket IMO. From my understanding of the discussion on Slack, @gpartenet mentioned the necessity to provide a separate, _independent_ scope value from what is configured on the provider in the console. Here, you can only widen the configured scope rather than provide another one.